src/sqlfluff/core/errors.py

```diff
@@ -219,6 +219,8 @@ def check_tuple(self):
         )
 
     def __repr__(self):
-        return "<SQLLintError: rule {0} pos:{1!r}, #fixes: {2}>".format(
-            self.rule_code(), self.pos_marker(), len(self.fixes)
+        return (
+            "<SQLLintError: rule {0} pos:{1!r}, #fixes: {2}, #description: {3}>".format(
+                self.rule_code(), self.pos_marker(), len(self.fixes), self.description

```

Comment:
just added the description here , it was useful when I was logging but let me know if it doesn't make sense to add

Reply:
it uses `#fixes:` because it's counting the number of fixes. For the description field I think it should be just `description: ` rather than `#description: `.